ts/features/wallet/cobadge/component/BaseCoBadgeCard.tsx

```diff
@@ -67,6 +68,10 @@ const BaseCoBadgeCard: React.FunctionComponent<Props> = (props: Props) => {
       resizeMode: "contain"
     })
   );
+  const isCobadgeExpired = isExpired(
+    props.expiringDate ? props.expiringDate.getMonth() - 1 : undefined,
+    props.expiringDate?.getFullYear()
+  ).getOrElse(false);
```

Comment:
The same of the previous comment.

Reply:
https://github.com/pagopa/io-app/pull/2982#discussion_r616576915